Plugins/UnityClientPlugin/WebRtcWrapper/Signalling/Signalling.cs

```diff
@@ -316,6 +316,41 @@ private void Close()
             _state = State.NOT_CONNECTED;
         }
 
+        /// <summary>
+        /// Updates the capacity data on the signaling server
+        /// </summary>
+        /// <remarks>
+        /// Once capacity reaches 0, we'll be removed from the signaling server listing of available peers
+        /// see https://github.com/bengreenier/webrtc-signal-http-capacity for more info
+        /// </remarks>
+        /// <param name="newCapacity">the new remaining capacity</param>
+        /// <returns>success flag</returns>
+        public async Task<bool> UpdateCapacity(int newCapacity)
+        {
+            if (_state != State.CONNECTED)
+            {
+                return false;
+            }
+
+            Debug.Assert(IsConnected());
+
+            if (!IsConnected())
+            {
+                return false;
+            }
+

```

Comment:
Do we need both conditions: `_state != State.CONNECTED` and `!IsConnected()`? Seem to me that *_myId* is always equal to -1 when *_state* is equal to *State.NOT_CONNECTED*.

Reply:
removed one